app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  # Log in the given user
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  # return current logged in user
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  # return true if the user is logged in
+  def logged_in?
+    !current_user.nil?
```

Comment:
@onlyno2 
Có thể dùng present?
```suggestion
    current_user.present?
```